app/views/paginable/plans/_org_admin.html.erb

```diff
@@ -29,9 +29,9 @@
         <tr>
           <td>
             <% if plan.readable_by?(current_user.id) %>
-              <%= link_to "#{plan.title.length > 60 ? "#{plan.title[0..59]} ..." : plan.title}", plan_path(plan) %>
+              <%= link_to sanitize("#{plan.title.length > 60 ? "#{plan.title[0..59]} ..." : plan.title}"), plan_path(plan) %>
             <% else %>
-              <%= plan.title.truncate(60) %>
+              <%= sanitize plan.title.truncate(60) %>
```

Comment:
We should use parenthesis here as well. There are a few cases where this happens.

```
 sanitize(plan.title.truncate(60))
```